tests/src/test/resources/junit-platform.properties

```diff
@@ -1,5 +1,5 @@
 # Timeout config
-junit.jupiter.execution.timeout.testable.method.default=10 s
+junit.jupiter.execution.timeout.testable.method.default=15 s
```

Comment:
This timeout has been increased by 5 seconds because it can take up to 5s until the new schema information has been propagated from the schema registry to the ingress.

Reply:
This is gonna kill the execution time of tests... Could we have a way to significantly lower down the polling time, to say 100ms, for tests just by configuring an env variable here https://github.com/restatedev/e2e/blob/main/tests/build.gradle.kts#L51?

Reply:
Also isn't 5 seconds generally too much? Also for the local dev experience...

Reply:
I've added a commit which eagerly pushes schema information changes to the worker. This will make the schema information schema propagation appear almost instantaneously. Will revert this change then.